### PR TITLE
TENET

### PR DESCRIPTION
**Pull request by hargunoberoi**
What's happened happened!

tenet

### CODE REVIEW

File: ITPMadeYouDoWhat.md [hunk 1]
```diff
@@ -16,4 +16,6 @@ I ate three apples today!!
 
 and mangoes are aaamaazing
 
-I ate a jalapeno cheddar stick today.
\ No newline at end of file
+I ate a jalapeno cheddar stick today.
```
Comment (Rushali) [inline ITPMadeYouDoWhat.md:19]: amazing

File: ITPMadeYouDoWhat.md [hunk 2]
```diff
@@ -16,4 +16,6 @@ I ate three apples today!!
...
+
+I asked Shiffman for stickers
```
Comment (Rushali) [inline ITPMadeYouDoWhat.md:21]: and you got them!

**Review by Rushali — APPROVED** [review]
Im approvin your changes! good work

**Comment by hargunoberoi** [comment]
Thank you!

**Comment by hargunoberoi** [comment]
Thank you. You can now watch TENET in reverse.

**Pull request merged**